src/getDefaultHostingSite.ts

```diff
@@ -0,0 +1,19 @@
+import * as logger from "./logger";
+import { getFirebaseProject } from "./management/projects";
+import * as _ from "lodash";
+
+/**
+ * Tries to determine the default hosting site for a project, else falls back to projectId.
+ * @param {Object} options The command-line options object
```

Comment:
In typescript, types are not necessary in jsdoc comments.

```suggestion
 * @param options The command-line options object
```